Biasing/src/Biasing/TargetBremFilter.cxx

```diff
@@ -85,6 +85,8 @@ void TargetBremFilter::stepping(const G4Step* step) {
    */
   if (auto volume{track->GetNextVolume()->GetName()};
       volume.compareTo("recoil_PV") == 0
+//      volume.compareTo("recoil_PV") == 0)
+//      volume.compareTo("recoil") == 0)
```

Comment:
similarly here I dont know how `volume.compareTo("recoil")` would differ from `olume.compareTo("recoil_PV")`, but I think these two lines dont need to be included anymore

Reply:
The Geant4 GDML parser adds the `_PV` suffix and `compareTo` requires a full match (not just a substring), so using the full name `recoil_PV` is necessary here. You are correct, we could remove the extraneous comments for code cleanliness.

Reply:
Resolved in https://github.com/LDMX-Software/ldmx-sw/pull/1285/commits/20a7ceb54936afaf433a5e6b2ccfda606e99d4b3